Gemfile.lock

```diff
@@ -1,58 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.3.3)
-      actionview (= 6.0.3.3)
-      activesupport (= 6.0.3.3)
-      rack (~> 2.0, >= 2.0.8)
+    actionpack (6.1.3)
+      actionview (= 6.1.3)
+      activesupport (= 6.1.3)
+      rack (~> 2.0, >= 2.0.9)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.3.3)
-      activesupport (= 6.0.3.3)
+    actionview (6.1.3)
+      activesupport (= 6.1.3)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.3.3)
-      activesupport (= 6.0.3.3)
-    activerecord (6.0.3.3)
-      activemodel (= 6.0.3.3)
-      activesupport (= 6.0.3.3)
-    activesupport (6.0.3.3)
+    activemodel (6.1.3)
+      activesupport (= 6.1.3)
+    activerecord (6.1.3)
+      activemodel (= 6.1.3)
+      activesupport (= 6.1.3)
+    activesupport (6.1.3)
       concurrent-ruby (~> 1.0, >= 1.0.2)
-      i18n (>= 0.7, < 2)
-      minitest (~> 5.1)
-      tzinfo (~> 1.1)
-      zeitwerk (~> 2.2, >= 2.2.2)
+      i18n (>= 1.6, < 2)
+      minitest (>= 5.1)
+      tzinfo (~> 2.0)
+      zeitwerk (~> 2.3)
     builder (3.2.4)
-    concurrent-ruby (1.1.7)
+    concurrent-ruby (1.1.8)
     crass (1.0.6)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
-    erubi (1.9.0)
-    i18n (1.8.5)
+    erubi (1.10.0)
+    i18n (1.8.9)
       concurrent-ruby (~> 1.0)
-    json (2.3.1)
+    json (2.5.1)
     jwt (2.2.2)
-    jwt_signed_request (2.6.0)
+    jwt_signed_request (3.0.0)
       jwt (>= 1.5.0)
       rack
-    loofah (2.7.0)
+    loofah (2.9.0)
```

Comment:
### [loofah](https://github.com/flavorjones/loofah)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.7.0 → 2.9.0

[_[change-log](https://github.com/flavorjones/loofah/blob/master/CHANGELOG.md), [source-code](https://github.com/flavorjones/loofah)_]





<details>
<summary>Commits</summary>

A change of **22** commits. See the full changes on [the compare page](https://github.com/flavorjones/loofah/compare/v2.7.0...v2.9.0).

These are the first 10 commits:
- (d514ca5) [Create FUNDING.yml](https://github.com/flavorjones/loofah/commit/d514ca5f2b5e9d950bd038933ec4c873c018e31c)
- (7143893) [README: add tidelift link](https://github.com/flavorjones/loofah/commit/7143893beb8e41a687cf5085145dd7d90ba23881)
- (b892fa6) [README: badges no longer in table format, add tidelift badge](https://github.com/flavorjones/loofah/commit/b892fa64e6cd47de8c6853c3739ba6337e257134)
- (f793fe8) [extract Loofah::VERSION into loofah/version.rb](https://github.com/flavorjones/loofah/commit/f793fe8b879e7e504d1b3b0bbac2c1f777b00cc7)
- (a530d32) [drop Hoe and use a gemspec file instead](https://github.com/flavorjones/loofah/commit/a530d32926e551b526d4b10aa8bae3eb31429268)
- (f381722) [ci: prefer registry-image to docker-image](https://github.com/flavorjones/loofah/commit/f381722f0c345409b71c6b556e4f5f295de83170)
- (7f3a047) [fix the :test rake task](https://github.com/flavorjones/loofah/commit/7f3a047b99452d714da3a6a8af40173510d661a1)
- (6429526) [test: fix warnings](https://github.com/flavorjones/loofah/commit/642952606db8fdfadf51cd4de6966f9224062c87)
- (1499384) [ci: upgrade to teliaoss/github-pr-resource](https://github.com/flavorjones/loofah/commit/149938466aa94ae6e03efe6f8c18d80e1329a092)
- (7aeeda5) [Update rubocop requirement from ~> 0.89 to ~> 1.1](https://github.com/flavorjones/loofah/commit/7aeeda5f88c7eca7cc45729146ab3fd9c9cfd959)


</details>